sp800-63-3/errata.md

```diff
@@ -0,0 +1,17 @@
+<div class="breaker"></div>
+<a name="errata"></a>
+
+<div class="text-center" markdown="1">
+## Errata
+</div> 
+
+This table contains changes that have been incorporated into Special Publication 800-63-3. Errata updates can include corrections, clarifications, or other minor changes in the publication that are either editorial or substantive in nature.
+
+|Date|Type|Change|Location
+|----|----|----|----|
+|2017-12-01|Editorial|Removed the term ‘cryptographic’ from the AAL3 description.|Executive Summary|
```

Comment:
Please change to "straight" single quotes

Reply:
Copy-paste error from the PDF, thanks.